src/gpt_computer_agent/config/settings.py

```
@@ -0,0 +1,113 @@
+    API_PREFIX: str = "/api"
+    BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = ["http://localhost:8000"]
+    
+    # Authentication
+    ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 8  # 8 days
+    REFRESH_TOKEN_EXPIRE_DAYS: int = 30
+    JWT_ALGORITHM: str = "HS256"
+    JWT_ISSUER: str = "gpt-computer-agent"
+    
+    # First user (admin) - for initial setup
+    FIRST_SUPERUSER: EmailStr = "admin@example.com"
+    FIRST_SUPERUSER_PASSWORD: str = "changeme"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "FIRST_SUPERUSER_PASSWORD"

(S105)